helm/chart/chartmuseum/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
As in other comment, I think we agreed that instead of the full chart, we should have a minimal chart.